iohk-monitoring/src/Cardano/BM/Configuration.lhs

```diff
@@ -66,7 +67,7 @@ testSeverity :: CM.Configuration -> LoggerName -> LOMeta -> IO Bool
 testSeverity config loggername meta = do
     globminsev  <- CM.minSeverity config
     globnamesev <- CM.inspectSeverity config loggername
-    let minsev = max globminsev $ fromMaybe Debug globnamesev
+    let minsev = globminsev <> fold globnamesev
```

Comment:
Hmm, you argued that `<>` should be `min`, but here the previous code was clearly `max`, right?

Reply:
Indeed, this was a :bug: